Gems/ROS2/Code/Source/Manipulation/JointsManipulationEditorComponent.cpp

```diff
@@ -108,8 +108,11 @@ namespace ROS2
                 azrtti_cast<ROS2::ROS2FrameEditorComponent*>(Utils::GetGameOrEditorComponent<ROS2::ROS2FrameEditorComponent>(entity));
             AZ_Assert(frameEditorComponent, "ROS2FrameEditorComponent does not exist!");
 
+            const bool hasArticulations = Utils::CheckIfEntityHasComponentOfType(
```

Comment:
Ideally this would be in a typeid header - but moving it out in the O3DE makes sense as a separate effort